repairnator/docker-images/bears-dockerimage/logback.xml

```diff
@@ -14,15 +14,15 @@
 	</appender>
 
 	<logger name="fr.inria.spirals.jtravis.helpers" level="info" />
```

Comment:
It should be changed to `fr.inria.jtravis`

Reply:
Done.